src/components/select/option/option.component.js

```diff
@@ -31,6 +39,11 @@ Option.propTypes = {
   children: PropTypes.node,
   /** The option's invisible internal value */
   value: PropTypes.oneOfType([PropTypes.object, PropTypes.string]).isRequired,
+  /**
+   * @private
+   * @ignore
+   * Callback to return value when the element is clicked (prop added by the SelectList component) */
+  onClick: PropTypes.func.isRequired,
```

Comment:
This shouldn't be a required prop, users won't even know about it because it's private

Reply:
Yeah :D I wanted to mark it as important for us, developers. But yeah, it makes non sense